<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartCfgDialog</class>
 <widget class="QDialog" name="PartCfgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Partition Configuration</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Preserve some space at the end of the disk</string>
    </property>
    <property name="autoRepeatInterval">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>14</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>MB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GB</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Align partitions with 4KB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
